README.md

```diff
@@ -528,8 +528,9 @@ df["Profit Percent"]=np.where(df['Category']=='Furniture',df['Per Units Sales']*
 df['visits_category'] = np.where(df['visits_30days']== 0, 'YES', 'NO')
 ```
 
-### Some more operations
-```python
+## Some more operations
```

Comment:
From here, I had separated the sub-heading and py code so as to make it more readable